src/topp/XTandemAdapter.cpp

```diff
@@ -432,7 +432,7 @@ class TOPPXTandemAdapter :
     search_parameters.fixed_modifications = getStringList_("fixed_modifications");
     search_parameters.variable_modifications = getStringList_("variable_modifications");
     search_parameters.missed_cleavages = getIntOption_("missed_cleavages");
-    search_parameters.peak_mass_tolerance = getDoubleOption_("fragment_mass_tolerance");
+    search_parameters.fragment_mass_tolerance = getDoubleOption_("fragment_mass_tolerance");
```

Comment:
shouldnt we also set
```
bool fragment_mass_tolerance_ppm;
bool precursor_mass_tolerance_ppm;
```
using the XTandemAdapter parameters now that its available?

other issue:
"precursor_mass_tolerance" in the adapter is currently "1.5", but in ppm. This seems a rather stringent error windows and was probably ok for Da earlier. Could you increase that to ~20ppm?!